ajax/libs/maptalks/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "maptalks",
+  "filename": "maptalks.min.js",
+  "version": "0.37.0",
+  "description": "A light JavaScript library to create integrated 2D/3D maps.",
+  "license": "BSD-3-Clause",
+  "author": "Fu Zhen",
+  "keywords": [
+    "javascript",
+    "gis",
+    "map",
+    "2d-maps",
+    "3d-maps",
+    "geospatial",
+    "maps",
+    "maptalks"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/maptalks/maptalks.js.git"
+  },
+  "homepage": "https://maptalks.org",
+  "npmName": "maptalks",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "**/*"
```

Comment:
Hey, @finch185277 did you remember how to use auto-updater?
After you finish the pattern, you can try `./auto-update.js run maptalks` to check everything grab the right files. (you can use the command `tree` & `du` ... etc to help you check file structure)

**This patten should be fixed.** Would you try to exclude unnecessary files which can't be used in web front-end by rewrite the pattern? Like `ajax/libs/maptalks/0.1.0/maptalks.min.js.gz`, `ajax/libs/maptalks/0.6.0/canvg`, `ajax/libs/maptalks/0.7.0/api/*.html`, `ajax/libs/maptalks/0.26.3/images/control/Thumbs.db`, `ajax/libs/maptalks/0.30.2/LICENSE`, `ajax/libs/maptalks/0.30.2/ACKNOWLEDGEMENT` ... etc

Also, if you find some file name include a version number, you have to try to manually add them.
Besides, can `maptalks.es.js` be used in web front-end?

If you don't know which file needs to be hosted, you could open an issue on upstream's git repo to discuss the author with the problem you met.

Reply:
I can use 
```
      "files": [
        "**/!(ACKNOWLEDGEMENT|LICENSE|*.es.js|*.gz)"
      ]
```
to shrink the scope.
But how to restrict `images/control/Thumbs.db`, `canvg/*`, `api/*.html`?
@sufuf3  Would you like to show me some examples?